easybuild/easyconfigs/m/MLC/MLC-3.0.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'Tarball'
+
+name = 'MLC'
+version = '3.0'
+
+homepage = 'https://software.intel.com/en-us/articles/intelr-memory-latency-checker'
+description = """Intel Memory Latency Checker (Intel MLC) is a tool used to measure memory latencies and b/w,
+ and how they change with increasing load on the system."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# download requires agreeing to license
+# https://software.intel.com/en-us/articles/intelr-memory-latency-checker#download
+sources = ['mlcv3.0.tgz']
+
+sanity_check_paths = {
+    'files': ['Linux/mlc', 'Linux/mlc_avx512', 'mlc_license.txt', 'readme.pdf'],
```

Comment:
well, they're part of the tarball being unpacked... if they're not there, soemthing is wrong

and we do this in other places too, license and documentation are important part of an installation?